stetho/src/main/java/com/facebook/stetho/common/android/ViewUtil.java

```diff
@@ -14,6 +19,85 @@ private ViewUtil() {
   }
 
   @Nullable
+  public static View hitTest(View view, float x, float y) {
+    return hitTest(view, x, y, null);
+  }
+
+  // x,y are in view's local coordinate space
```

Comment:
Javadoc.  Also, what does "local coordinate space" mean?  Is it relative to the view's `getLeft()/getTop()`?  Be specific please.